system/autoware_default_adapi/src/planning.hpp

```diff
@@ -30,22 +35,53 @@
 namespace autoware::default_adapi
 {
 
+using autoware_adapi_v1_msgs::msg::PlanningBehavior;
+using autoware_adapi_v1_msgs::msg::SteeringFactor;
+using autoware_adapi_v1_msgs::msg::SteeringFactorArray;
+using autoware_adapi_v1_msgs::msg::VelocityFactor;
+using autoware_adapi_v1_msgs::msg::VelocityFactorArray;
+using tier4_planning_msgs::msg::PlanningFactor;
+using tier4_planning_msgs::msg::PlanningFactorArray;
+
+const std::map<std::uint16_t, std::uint16_t> direction_map = {
+  {PlanningFactor::SHIFT_RIGHT, SteeringFactor::RIGHT},
+  {PlanningFactor::SHIFT_LEFT, SteeringFactor::LEFT},
+  {PlanningFactor::TURN_RIGHT, SteeringFactor::RIGHT},
+  {PlanningFactor::TURN_LEFT, SteeringFactor::LEFT}};
+
+const std::map<std::string, std::string> conversion_map = {
+  {"static_obstacle_avoidance", PlanningBehavior::AVOIDANCE},
+  {"crosswalk", PlanningBehavior::CROSSWALK},
+  {"goal_planner", PlanningBehavior::GOAL_PLANNER},
+  {"intersection", PlanningBehavior::INTERSECTION},
+  {"lane_change_left", PlanningBehavior::LANE_CHANGE},
+  {"lane_change_right", PlanningBehavior::LANE_CHANGE},
+  {"merge_from_private", PlanningBehavior::MERGE},
+  {"no_stopping_area", PlanningBehavior::NO_STOPPING_AREA},
+  {"blind_spot", PlanningBehavior::REAR_CHECK},
+  {"obstacle_cruise_planner", PlanningBehavior::ROUTE_OBSTACLE},
+  {"motion_velocity_planner", PlanningBehavior::ROUTE_OBSTACLE},
+  {"walkway", PlanningBehavior::SIDEWALK},
+  {"start_planner", PlanningBehavior::START_PLANNER},
+  {"stop_line", PlanningBehavior::STOP_SIGN},
+  {"surround_obstacle_checker", PlanningBehavior::SURROUNDING_OBSTACLE},
+  {"traffic_light", PlanningBehavior::TRAFFIC_SIGNAL},
+  {"detection_area", PlanningBehavior::USER_DEFINED_DETECTION_AREA},
+  {"virtual_traffic_light", PlanningBehavior::VIRTUAL_TRAFFIC_LIGHT},
+  {"run_out", PlanningBehavior::RUN_OUT}};
```

Comment:
These variables can be moved to cpp file.

Reply:
Fixed in https://github.com/tier4/autoware.universe/pull/1734/commits/615d1bb7371048c3c0bf5a71f067a92e378c8d9e.